src/sage/graphs/graph.py

```diff
@@ -7518,8 +7546,20 @@ def cliques_containing_vertex(self, vertices=None, cliques=None):
             sage: sorted(sorted(x for x in L) for L in d[(0, 1)])
             [[(0, 0), (0, 1)], [(0, 1), (0, 2)], [(0, 1), (1, 1)]]
         """
-        import networkx
-        return networkx.cliques_containing_node(self.networkx_graph(), vertices, cliques)
+        if cliques is None:
+            cliques = self.cliques_maximal()
+
+        if vertices in self:  # single vertex
+            return [c for c in cliques if vertices in c]
+
+        from collections import defaultdict
+        d = defaultdict(list)
+
+        for c in cliques:
+            for v in c:
+                d[v].append(c)
+
+        return { v : d[v] for v in vertices or self }
```

Comment:
coding style: `return { v : count[v] for v in vertices or self }` -> `return {v : count[v] for v in vertices or self}`

Reply:
Done.